algorithms/corpus_creators/basic.py

```diff
@@ -30,6 +30,21 @@ def raw_corpora(self, generator):
             document = json.loads(line)
             yield self._transform(document)
 
+    def array_corpora(self, generator):
+        """
+        """
+        for line in generator:
+            document = json.loads(line)
+            #print(self._transform(document).split())
+            yield self._transform(document).split()
+
+    def label_corpora(self,generator):
```

Comment:
This is especially important to create a docstring for, because currently they're fake labels.